contrib/compose/example-app/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM debian
```

Comment:
3.) to create a new client on the fly. See `contrib/compose/example-app/run.sh` better would be to have the state-outside, but then I need to pass in the data during startup. At this point, dex is not up and running... Any proposals to improve this?